server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader12to13.java

```diff
@@ -438,4 +447,50 @@ void addTableMappingsToZooKeeper(ServerContext context) {
           "Could not read or write metadata in ZooKeeper because of ZooKeeper exception", ex);
     }
   }
+
+  void moveTableProperties(ServerContext context) {
+    try {
+      final SystemPropKey spk = SystemPropKey.of();
+      final VersionedProperties sysProps = context.getPropStore().get(spk);
+      final Map<String,String> sysTableProps = new HashMap<>();
+      sysProps.asMap().entrySet().stream()
+          .filter(e -> e.getKey().startsWith(Property.TABLE_PREFIX.getKey()))
+          .forEach(e -> sysTableProps.put(e.getKey(), e.getValue()));
+      LOG.info("Adding the following table properties to namespaces unless overridden, {}",
```

Comment:
logging an entire map in one line can be hard to read in the logs when its a large map, could log one line per prop

Reply:
Fixed in f452cae.